charters/DataPortal/charter.md

```diff
@@ -0,0 +1,69 @@
+# Data Portal
+
+
+## Description
+The Data Portal is the landing site for the Human Cell Atlas (HCA) Data Coordination Platform (DCP). It is hosted at https://www.data.humancellatlas.org/ with an access link from the main HCA website at https://www.humancellatlas.org/. It provides the user experience (UX) for users to explore cellular resolution data.
+
+## Definitions
+*DCP* The Data Coordination Platform is the name given to the entire system used to ingest, validate, store, analyzes, and make available the data in the Human Cell Atlas project.
+
+*Data Browser* A DCP application which provides a faceted search capability.
+
+## Objectives
+The objective of the Data Portal is to provide the DCP website implementation. It provides the access point for users to search all HCA genetic data in order to identify a subset of the data for download and/or further analysis at petabyte scale on public clouds.
+
+## In-scope
+
+### Core capabilities
+* Implement a UX, visual, and branding design with input from the HCA brand working group.
+* Maintain a content management system that meets the needs of Data Portal content developers.
+* Integrate the Data Browser User Interface (UI) and Data Portal UI into a seamless UX and provide links to other DCP applications (e.g. the Ingestion Service UI).
```

Comment:
I think the distinction between browser and other applications is artificial. Why not just "Integrate DCP applications (e.g. the Data Browser, Ingestion Service) into a seamless UX with a common look and feel"?

Reply:
Agreed.